src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -51,47 +51,62 @@
  * is what should be used to enforce strict camel casing. The identifier 'MyTest' would
  * be allowed, but 'MyTEst' would not be.
  * </p>
+ * <ul>
+ * <li>
+ * Option {@code allowedAbbreviationLength} - indicates on the number of consecutive capital
+ * letters allowed in targeted identifiers (abbreviations in the classes, interfaces, variables
+ * and methods names, ... ). Default value is {@code 3}.
+ * </li>
+ * <li>
+ * Option {@code allowedAbbreviations} - list of abbreviations that must be skipped for checking.
+ * Abbreviations should be separated by comma. Default value is {@code {}}.
+ * </li>
+ * <li>
+ * Option {@code ignoreFinal} - allow to skip variables with {@code final} modifier. Default value
+ * is {@code true}.
+ * </li>
+ * <li>
+ * Option {@code ignoreStatic} - allow to skip variables with {@code static} modifier. Default
+ * value is {@code true}.
+ * </li>
+ * <li>
+ * Option {@code ignoreOverriddenMethods} - Allows to ignore methods tagged with {@code @Override}
+ * annotation (that usually mean inherited name). Default value is {@code true}.
+ * </li>
+ * <li>
+ * Option {@code tokens} - tokens to check Default value is:
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CLASS_DEF">CLASS_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">INTERFACE_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ENUM_DEF">ENUM_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_DEF">ANNOTATION_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_FIELD_DEF">ANNOTATION_FIELD_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#PARAMETER_DEF">PARAMETER_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#VARIABLE_DEF">VARIABLE_DEF</a>,
+ * <a href="http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">METHOD_DEF</a>.}.
+ * </li>
```

Comment:
What about
http://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html
?

Reply:
What do you mean by this question ?

Reply:
@romani He is saying we should change links from `http://checkstyle.sourceforge.net` to `http://checkstyle.org` which I agree with. Code of PR was started long ago, hence why it is coded like this.

I am waiting on Issue approval before fixing as CI will still fail since it isn't approved.

Reply:
Done.

Reply:
Sorry, I didn't see your message till now. I will undo.